lib/src/l10n/localizations.dart

```diff
@@ -114,42 +116,84 @@ class AppLocalizations {
 
   // Core
   String get coreChatNoMessages => Intl.message('No messages', name: 'coreChatNoMessages');
+
   String get coreSelf => Intl.message('Me', name: 'coreSelf');
+
   String get coreDraft => Intl.message('Draft', name: 'coreDraft');
+
   String get coreMembers => Intl.message('%1\$d member(s)', name: 'createChatWith');
+
   String get coreContacts => Intl.message('%1\$d contact(s)', name: 'createChatWith');
+
   String get coreVoiceMessage => Intl.message('Voice message', name: 'coreVoiceMessage');
+
   String get coreContactRequest => Intl.message('Contact request', name: 'coreContactRequest');
+
   String get coreImage => Intl.message('Image', name: 'coreImage');
+
   String get coreVideo => Intl.message('Video', name: 'coreVideo');
+
   String get coreAudio => Intl.message('Audio', name: 'coreAudio');
+
   String get coreFile => Intl.message('File', name: 'coreFile');
+
   String get coreGroupHelloDraft => Intl.message('Hello, I\'ve just created this group for us', name: 'coreGroupHelloDraft');
+
   String get coreGroupNameChanged => Intl.message('Group name changed', name: 'coreGroupNameChanged');
+
   String get coreGroupImageChanged => Intl.message('Group image changed', name: 'coreGroupImageChanged');
+
   String get coreGroupMemberAdded => Intl.message('Member added', name: 'coreGroupMemberAdded');
+
   String get coreGroupMemberRemoved => Intl.message('Member removed', name: 'coreGroupMemberRemoved');
+
   String get coreGroupLeft => Intl.message('Group left', name: 'coreGroupLeft');
+
   String get coreGenericError => Intl.message('Error: ', name: 'coreGenericError');
+
   String get coreGif => Intl.message('Gif', name: 'coreGif');
-  String get coreMessageCannotDecrypt => Intl.message('This message cannot be decrypted.\n\n'
+
+  String get coreMessageCannotDecrypt => Intl.message(
+      'This message cannot be decrypted.\n\n'
       '• It might already help to simply reply to this message and ask the sender to send the message again.\n\n'
-      '• In case you re-installed Delta Chat or another email program on this or another device you may want to send an Autocrypt Setup Message from there.', name: 'coreMessageCannotDecrypt');
+      '• In case you re-installed Delta Chat or another email program on this or another device you may want to send an Autocrypt Setup Message from there.',
```

Comment:
```suggestion
      '• In case you re-installed OX Talk or another email program on this or another device you may want to send an Autocrypt Setup Message from there.',
```
I think this is better